src/stylus/generic/_transitions.styl

```diff
@@ -47,7 +47,7 @@
     opacity: 0
 
   &-enter-to
-    transtion: translate(0, 0)
+    transition: translate(0, 0)
```

Comment:
This is still invalid, looks like it's actually supposed to be `transform`. Check if this is even needed at all, it seems to work fine as is.

Reply:
😲 Oops. I will update my PR at least for sake of code consistency…